Tests/Security/FormAuthenticatorTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Tests\Security;
+
+use PHPUnit_Framework_TestCase;
+use SumoCoders\FrameworkMultiUserBundle\Security\FormAuthenticator;
+use SumoCoders\FrameworkMultiUserBundle\Security\FormCredentials;
+use SumoCoders\FrameworkMultiUserBundle\Security\ObjectUserProvider;
+use SumoCoders\FrameworkMultiUserBundle\User\InMemoryUserRepository;
+use SumoCoders\FrameworkMultiUserBundle\User\User;
+use Symfony\Bundle\FrameworkBundle\Routing\Router;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\HttpFoundation\Session\Storage\MockArraySessionStorage;
+use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
+use Symfony\Component\Security\Core\Encoder\EncoderFactory;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoder;
+
+class FormAuthenticatorTest extends PHPUnit_Framework_TestCase
+{
+    private $formAuthenticator;
+    private $router;
+
+    public function setUp()
+    {
+        $this->router = new Router(new Container(), '');
```

Comment:
Dan krijg je ```Cannot mock Symfony\Component\Routing\Router::addExpressionLanguageProvider() because a class or interface used in the signature is not loaded```